example-get-started-experiments/generate.sh

```diff
@@ -67,56 +71,45 @@ tick
 git commit -m "Initialize DVC project"
 
 
-cp -r $HERE/code/data .
-git add data/.gitignore data/pool_data.dvc
+cp -r $HERE/code/datasets .
+git add datasets/.gitignore datasets/pool_data.dvc
 tick
 git commit -m "Add data"
 dvc pull
 
 
-cp -r $HERE/code/notebooks .
+cp -r $HERE/code/TrainSegModel.ipynb .
 git add .
 git commit -m "Add notebook using DVCLive"
 
+sudo apt-get update && sudo apt-get install ffmpeg libsm6 libxext6 -y
 pip install -r requirements.txt --extra-index-url https://download.pytorch.org/whl/cu118
 pip install jupyter
-jupyter nbconvert --execute 'notebooks/TrainSegModel.ipynb' --inplace
-# Apply best experiment
-BEST_EXP_ROW=$(dvc exp show --drop '.*' --keep 'Experiment|evaluate/dice_multi|base_lr' --csv --sort-by evaluate/dice_multi | tail -n 1)
-BEST_EXP_NAME=$(echo $BEST_EXP_ROW | cut -d, -f 1)
-BEST_EXP_BASE_LR=$(echo $BEST_EXP_ROW | cut -d, -f 3)
-dvc exp apply $BEST_EXP_NAME
+yolo settings datasets_dir="/workspaces/example-repos-dev/example-get-started-experiments/build/example-get-started-experiments/datasets/"
+yolo settings runs_dir="/workspaces/example-repos-dev/example-get-started-experiments/build/example-get-started-experiments/runs/"
+yolo settings weights_dir="/workspaces/example-repos-dev/example-get-started-experiments/build/example-get-started-experiments/weights/"
+jupyter nbconvert --execute 'TrainSegModel.ipynb' --inplace
 git add .
 tick
 git commit -m "Run notebook and apply best experiment"
```

Comment:
Is it still applying the best experiment?

Reply:
No, do you think we should keep that thing? I am not even sure if people realize what was going one without checking this script

Reply:
No, I just meant we should change the commit message